examples/analog_in/cont_voltage_acq_int_clk.py

```diff
@@ -0,0 +1,31 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to acquire a continuous amount of data
+using the DAQ device's internal clock.
+"""
+
+import pprint
+import time
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+pp = pprint.PrettyPrinter(indent=4, compact=True)
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(
+        1000.0, sample_mode=AcquisitionType.CONTINUOUS, samps_per_chan=50
+    )
+    print("Starting task. Press Ctrl+C to stop.")
+    time.sleep(1.0)
+    task.start()
+
+    try:
+        while True:
+            data = task.read(number_of_samples_per_channel=10)
```

Comment:
(1000 S/s) / (10 S/read) = 100 read/s

Writing a new line to the console 100 times per second is too fast. Even 10 times per second is probably too fast, if it scrolls the console.

Either reduce the number of reads/sec to something like 1 or write to the console in a way that does not rapidly scroll the console. 

Do you understand what `ContAcq-IntClk-EveryNSamples.c` does?
```
printf("Acquired %d samples. Total %d\r",(int)read,(int)(totalRead+=read));
```

The `\r` is a carriage return, which returns the cursor to the leftmost column without advancing to the next line. Perhaps `print("...", end="\r")` or `print("...\r", end="")` would be useful for these two continuous Python examples.